client/src/components/SupermarketDisplay.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+function SupermarketDisplay({ supermarket }) {
+  return (
+    <Container>
```

Comment:
This is interesting. You are styling your whole component by applying all your styles to an outer container. Haven't seen this so far and was wondering about the advantages and disadvantages of doing it that way. @marwinburesch , can you comment on that?

Reply:
Maybe this could be discussed outside my PR? ;) I will merge it now.